framework/src/org/checkerframework/framework/util/MultiGraphQualifierHierarchy.java

```diff
@@ -234,12 +235,48 @@ protected void finish(QualifierHierarchy qualHierarchy,
     @SideEffectFree
     @Override
     public String toString() {
-        // TODO: it would be easier to debug if the graph and map were sorted by the key.
```

Comment:
The supertypes graph and supertypes map both call AnnotationUtils.createAnnotationMap(); to create its data structure. That method returns a TreeMap with alphabetical sort ordering.

I believe this TODO comment wasn't removed in a prior commit that made the changes to use the sorted TreeMap.

Current example output:
Note: Supertypes Graph: 
  	@org.checkerframework.checker.units.qual.A = [@org.checkerframework.checker.units.qual.Current]
  	@org.checkerframework.checker.units.qual.Acceleration = [@org.checkerframework.checker.units.qual.UnknownUnits]
  	@org.checkerframework.checker.units.qual.Angle = [@org.checkerframework.checker.units.qual.UnknownUnits]
  	@org.checkerframework.checker.units.qual.Area = [@org.checkerframework.checker.units.qual.UnknownUnits]
  	@org.checkerframework.checker.units.qual.C = [@org.checkerframework.checker.units.qual.Temperature]
  	@org.checkerframework.checker.units.qual.Current = [@org.checkerframework.checker.units.qual.UnknownUnits]
...